pkgs/by-name/m1/m1ddc/package.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, darwin
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "m1ddc";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "waydabber";
+    repo = "m1ddc";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-obs2qQvSkIDsWhCXJOF1Geiqqy19KDf0InyxRVod4hk=";
+  };
+
+  postPatch = ''
+    substituteInPlace sources/ioregistry.m \
+        --replace-fail kIOMainPortDefault kIOMasterPortDefault
+  '';
+
+  buildInputs = with darwin.apple_sdk_11_0.frameworks; [
+    CoreDisplay
+    Foundation
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp m1ddc $out/bin
+  '';
+
+  meta = {
+    description = ''
+      Control external displays using DDC/CI on Apple Silicon Macs
+    '';
+    homepage = "https://github.com/waydabber/m1ddc";
+    license = lib.licenses.mit;
+    mainProgram = "m1ddc";
+    maintainers = [ lib.maintainers.joanmassachs ];
+    platforms = [ "aarch64-darwin" ];
```

Comment:
does it only work with `aarch64-darwin`, maybe you could try `lib.platforms.darwin` and see how ofborg handle `x86-64_darwin`

Reply:
Well, the m1ddc readme says that it does not support Intel Macs, only Apple Silicon Macs. Even if ofborg managed to compile it, I imagine that the tool would not work